docs/pages/management/guides/docker.mdx

```diff
@@ -116,6 +122,14 @@ Create Teleport configs and start the process with the following `docker run` co
 # Create local config and data directories for Teleport, which will be mounted
 # into the container.
 $ mkdir -p ~/teleport/config ~/teleport/data
+```
+<Details title="Obtain License File" opened={false}>
```

Comment:
Since all Enterprise users will need this step, I think we should:

- Separate this `Tabs` component into three `TabItem`s, one for each edition
- In the Enterprise `TabItem`, include the instructions without a `Details` box
- Don't include license info in the `TabItem` for Teleport Cloud

Reply:
@ptgott are we looking to show running a docker container that connects to a Cloud tenant?  Otherwise it doesn't make sense to run a docker example with cloud.